backend/data_import/pipeline/data.py

```diff
@@ -46,6 +47,8 @@ def create(self, project: Project) -> Example:
             upload_name=self.upload_name,
             text=self.text,
             meta=self.meta,
+            article_id=self.article_id,
+            order=int(self.order),
         )
```

Comment:
Sorry for one more thing, I just tried running the tests and they gave an error `invalid literal for int() with base 10: ''`. Maybe we should change it so that if `self.order` is an empty string then order value should be NULL?

So something like `order = int(self.order) if self.order != "" else None` ? XD